docs/uploader/aws-s3-multipart.mdx

```diff
@@ -163,20 +169,49 @@ Drive). Otherwise it’s not worth the hosting effort.
 ```js {10} showLineNumbers
 import Uppy from '@uppy/core';
 import Dashboard from '@uppy/dashboard';
-import AwsS3Multipart from '@uppy/aws-s3-multipart';
+import AwsS3 from '@uppy/aws-s3';
 
 import '@uppy/core/dist/style.min.css';
 import '@uppy/dashboard/dist/style.min.css';
 
 const uppy = new Uppy()
 	.use(Dashboard, { inline: true, target: 'body' })
-	.use(AwsS3Multipart, { companionUrl: 'http://companion.uppy.io' });
+	.use(AwsS3, { companionUrl: 'http://companion.uppy.io' });
 ```
 
 ## API
 
 ### Options
 
+#### `shouldUseMultipart(file)`
+
+:::warning
+
+You must define this option, otherwise you’ll be using the
+[legacy version of this plugin](./aws-s3). If you use the deprecated
+`@uppy/aws-s3-multipart` package, this will be set to `true` by default, which
+is not a recomended setting.
```

Comment:
```suggestion
Not setting this option will result in a suboptimal experience for some of your user’s uploads.
It is best for speed and stability to upload large (100&nbsp;MB+) files with multipart
and small files with regular uploads.
```

Reply:
We probably want to keep the link to the legacy docs, no?

Reply:
```suggestion
Until the next major version, not setting this option uses the [legacy version of this plugin](./aws-s3).
This is a suboptimal experience for some of your user’s uploads.
It is best for speed and stability to upload large (100&nbsp;MB+) files with multipart
and small files with regular uploads.
```